store/postgres/asset_repository.go

```diff
@@ -25,27 +25,27 @@ type AssetRepository struct {
 }
 
 // GetAll retrieves list of assets with filters via config
-func (r *AssetRepository) GetAll(ctx context.Context, config asset.Config) ([]asset.Asset, error) {
-	size := config.Size
+func (r *AssetRepository) GetAll(ctx context.Context, cfg asset.Config) ([]asset.Asset, error) {
+	size := cfg.Size
 	if size == 0 {
 		size = r.defaultGetMaxSize
 	}
 
-	builder := r.getAssetSQL().
-		Limit(uint64(size)).
-		Offset(uint64(config.Offset))
-	builder = r.buildFilterQuery(builder, config)
+	builder := r.getAssetSQL().Limit(uint64(size)).Offset(uint64(cfg.Offset))
+	builder = r.buildFilterQuery(builder, cfg)
+	builder = r.buildOrderQuery(builder, cfg)
 	query, args, err := r.buildSQL(builder)
 	if err != nil {
 		return nil, fmt.Errorf("error building query: %w", err)
 	}
-	ams := []*AssetModel{}
+
+	var ams []*AssetModel
 	err = r.client.db.SelectContext(ctx, &ams, query, args...)
 	if err != nil {
 		return nil, fmt.Errorf("error getting asset list: %w", err)
 	}
 
-	assets := []asset.Asset{}
```

Comment:
we need this line `assets := []asset.Asset{}` to make GetAll to return empty slice instead of null if there are 0 records found.